### ISSUE TITLE
Should "Code of Conduct" include an agreement that LLMs will not be used for generating solutions? Or at least have a disclaimer that LLMs were used as part of the response?

### ISSUE THREAD

**Issue by cheapsteak** [getsentry/.github#119]
There are already a few instances of people using LLMs to generate very long, detailed, and wrong solutions to peoples' raised bugs
https://github.com/getsentry/sentry-javascript/issues/8285#issuecomment-1574758081
https://github.com/getsentry/sentry-javascript/issues/8281#issuecomment-1574306984

It's not immediately obvious that those were generated by LLM (even though it's quite glaring in hindsight), and it's quite exhausting to respond in a way that don't hurt their feelings while still communicating to anyone else (mainly the maintainers) who might just be skimming through that the large blob of confident-looking hallucination masquerading as a solution is actually _not_ a solution to the problem.

Thoughts on adding something to Code of Conduct that no responses should be generated via LLMs without proper disclaimers for the portions that are generated? And ideally also no LLM generated responses should be posted without the poster verifying that they really tried to understand both the originally posted problem and has done their best to verify the veracity of the generated text?

**Comment by cheapsteak** [comment]
Ah that makes sense wrt what the CoC is for 👍 

The reason I thought it might be good to have it spelled out somewhere is that it's not obvious that it's something you shouldn't do. I suspect @alikazi88 may have been trying to do something nice, like if you have an electric snow blower and help clear your neighbours' snow. 

Maybe a line in a github issue template would be more effective? But fair to wait to see if there will be a flood and make the change then. First time running into generated junk in the wild, I may be overreacting

**Comment by alikazi88** [comment]
I agree that I used LLM's to find a solution but in no way I wanted to create any sort of diversion or harassment to the person raising. To be honest it has been only a month since I started my coding journey and open-source contribution was something that was introduced to me. I had a very pious thought of helping, but seeing this reaction to the same, it actually demotivated me a lot and not it seems daunting. I hope one day I reach a level of excellency and still have the part of me where I mentor and not daunt a newbie without understanding the background.

**Comment by cheapsteak** [comment]
Hey @alikazi88, I figured your intentions were good (I did posit that [here](https://github.com/getsentry/.github/issues/119#issuecomment-1577045898)!), and that you don't yet understand why contributing to open source in the way that you did isn't helpful, but since you're here, perhaps you'd be interested in hearing out why someone might be so concerned as to open an issue like the one I did besides to daunt a newbie (and heck, maybe even get a kick out of it)  

When I initially read your very long and detailed answer in https://github.com/getsentry/sentry-javascript/issues/8281#issuecomment-1574306984 that I thought came from another human that spent a long time figuring out and putting together, I was elated and grateful. Once I realized you missed the point of the issue, I was bummed, but still grateful that you'd put that much time to make the attempt. Once you responded in https://github.com/getsentry/sentry-javascript/issues/8281#issuecomment-1574747934 to double down on the mistake with such confidence, I questioned myself, maybe I didn't understand or made a mistake. I read through everything again, I dug into this repo's code to see whether it's true that event ID is not meant to be accessible within the Sentry.ErrorBoundary. 

When I finally realized your responses were probably generated, honestly, I was pretty darned upset. I suspect I've spent significantly more time validating your responses than it took for you to generate them.   

But, this issue was not _at all_ drafted to target you or reprimand you, but rather out of fear of what's coming  

For as much as GPT has been in the news, it never occurred to me that someone might trawl through Github issues to transcribe what ChatGPT told them. Quite impressive if you came up with the idea, surprised quite a few folks in my local dev community. My worry is along the lines of après toi, le déluge.

If you've only been coding for a month, then perhaps you're too young to remember or realize that asking for people to be nice to each other online is a daunting task. The one mantra that kinder souls have tried to drill in to us is to "remember there's always a person on the other end of those wires". Terrifyingly, that's no longer true. Exercising empathy takes effort; how many people are going to bother when you're probably just responding to a language model anyway?  

Forgive the melodrama, but this might just be the death of empathy on the internet
